model/proto/http.proto

```diff
@@ -32,13 +33,14 @@ message HTTP {
 }
 
 message HTTPRequest {
+  reserved 4;
   google.protobuf.Value body = 1;
   repeated HTTPHeader headers = 2;
   repeated KeyValue env = 3;
-  repeated KeyValue cookies = 4;
```

Comment:
Should we keep and rename the old field, and convert to `map[string]string`? Otherwise we'll lose data?